### PR TITLE
Update OSS version from 17.1.0 to 18.1.0

### PR DESCRIPTION
**Pull request by mike0042**
#### Bug/feature Description
* The default version is being updated to 18.1.0 to match the year 2018
* No PP ticket

#### Affected Platform(s) and PBS Version
* All platforms that use the spec file and/or configure.ac

#### Cause / Analysis / Design
* The year is currently 2018

#### Solution Description
* Update the few files that contain a hard coded version starting with 17.

#### Checklist:

- [X] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [X] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [X] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
   - [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [ ] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
   - [ ] I have added  **manual test(s) to this pull request and explained why PTL is not appropriate** for this case.
- [ ] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [ ] I have attached **test logs to this pull request** as evidence of testing/verification.

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

### CODE REVIEW

File: INSTALL [hunk 1]
```diff
@@ -226,4 +226,4 @@ Here is a set of steps to detect memory errors/leaks within PBS code.
    export LD_LIBRARY_PATH=/opt/pbs/pgsql/lib:/opt/pbs/lib:$LD_LIBRARY_PATH
    valgrind --tool=memcheck --log-file=/tmp/val.out --suppressions=./valgrind.supp --leak-check=full --track-origins=yes /opt/pbs/sbin/pbs_server.bin
    
-  
\ No newline at end of file
+
```
Comment (agrawalravi90) [inline INSTALL]: Why the extra newline?

Comment (mike0042) [reply INSTALL]: No idea. Vim must have done it.

**Comment by mike0042** [comment]
@agrawalravi90 : Not sure what control characters were at the end of the INSTALL file. Must have been added with the valgrind instructions. The file now ends with a single blank line.

**Comment by agrawalravi90** [comment]
Thanks Mike, please squash and rebase for the final merge

**Comment by agrawalravi90** [comment]
Thanks Mike, I'll merge this in.

**Pull request merged**